app/src/main/java/com/example/marvelapp/data/local/model/utils/NumbersUtils.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.marvelapp.data.local.model.utils
```

Comment:
you can unify this class and the new StringUtils in a ContantsUtils

Reply:
Done, now they are in the seme file, thx!